packages/apps-config/src/endpoints/testing.ts

```diff
@@ -283,6 +282,17 @@ export const testChains: Omit<EndpointOption, 'teleport'>[] = [
       logo: chainsEquilibriumSVG
     }
   },
+  {
+    info: 'elysium',
+    providers: {
+      elysium: 'wss://ws.atlantischain.network'
+    },
+    text: 'Elysium',
+    ui: {
+      color: '#9FC131',
+      logo: nodesElysiusmPNG
```

Comment:
```suggestion
      logo: nodesElysiumPNG
```

typo (making this unknown and the import unused)